modules/roks_with_odf/README.md

```diff
@@ -0,0 +1,111 @@
+# IBM Red Hat OpenShift Managed Cluster Parameters and Installation
```

Comment:
This entire directory belongs in the https://github.com/ibm-hcbt/cloud-pak-sandboxes/tree/main/terraform repo.